screenpy_selenium/resolutions/custom_matchers/is_present_element.py

```diff
@@ -0,0 +1,44 @@
+"""
+A matcher that matches an element. For example:
+
+    assert_that(driver.find_element_by_id("search"), is_visible_element())
+"""
+from typing import Optional
+
+from hamcrest.core.base_matcher import BaseMatcher
+from hamcrest.core.description import Description
+from selenium.webdriver.remote.webelement import WebElement
+
+
+class IsPresentElement(BaseMatcher[Optional[WebElement]]):
+    """
+    Matches an element to be a WebElement.
+    """
+
+    def _matches(self, item: Optional[WebElement]) -> bool:
+        if item is None:
+            return False
+        return isinstance(item, WebElement)
+
+    def describe_to(self, description: Description) -> None:
+        """Describe the passing case."""
+        description.append_text("the element is present")
+
+    def describe_match(
+        self, item: Optional[WebElement], match_description: Description
+    ) -> None:
+        match_description.append_text("it was present")
+
+    def describe_mismatch(
+        self, item: Optional[WebElement], mismatch_description: Description
+    ) -> None:
+        """Describe the failing case."""
+        if item is None:
+            mismatch_description.append_text("was not even present")
```

Comment:
I think the "even" here should be left out. It was useful for the visibility matchers to indicate the visibility of the element couldn't be determined because the element wasn't even present. Here, it's the focus of the matcher, so i think just "was not present" is good.

Reply:
Which leaves the question of, should I just strip out the logic down to this: 
```
def describe_mismatch(
        self, item: Optional[WebElement], mismatch_description: Description
    ) -> None:
        """Describe the failing case."""
        mismatch_description.append_text("was not present")
```

Reply:
Ha, absolutely. :D

Reply:
updated!